web/src/scripts/containers/TabbedEditor.jsx

```diff
@@ -157,6 +157,11 @@ class TabbedEditor extends Component {
 
     // TODO consider also using command+t (which currently conflicts with a CM command)
     openFileSearch: 'command+p',
+    nextTab: 'command+]',
+    prevTab: 'command+[',
+    openInNewTab: 'command+alt+\\',
```

Comment:
For some reason, I can't trigger 'command+alt+\\'. No idea why. Maybe we do 'command+shift+\\'?